ReactNativeClient/lib/components/screens/note.js

```diff
@@ -372,6 +373,10 @@ class NoteScreenComponent extends BaseScreenComponent {
 			const resourceIds = await Note.linkedResourceIds(this.state.note.body);
 			await ResourceFetcher.instance().markForDownload(resourceIds);
 		}
+
+		if (this.state.note) {
+			addNoteToRecentsWidget(this.state.note);
```

Comment:
I suspect this might not work correctly if you create a new note and immediately go back without typing anything. The note will not be deleted (because it will not be saved in the first place) but looks like it will be put on this list.

Reply:
Good catch, you're right! I added a check to make sure the note has a title. If you have an idea on how this could be done more elegantly, let me know.

Reply:
I don't think it's currently possible to get the list of last opened notes, and if we add this it would be done differently, perhaps by re-using the note history logic. It seems simple to add but there's a ton of edge cases which would need to be supported by a large test suite (the history feature already does most of this).

So I feel that should be a for a different pull request. In the meantime, could you for example display the last modified notes? For this, we already have the logic in place so there will be less changes to make.